libs/blocks/merch/merch.js

```diff
@@ -302,11 +302,20 @@ async function openFragmentModal(path, getModal) {
   return modal;
 }
 
+function focusOnTab(url) {
+  const metaPreselectPlan = document.querySelector('meta[name="preselect-plan"]');
+  if (!metaPreselectPlan) return url;
+  const urlWithPlan = new URL(url);
```

Comment:
Is it certain that the `url` is valid? Otherwise a `try/catch` might be safer.

Reply:
good catch, the url is authored therefore could be invalid if GWP makes a mistake

Reply:
`try/catch` added